src/server/family_utils.h

```diff
@@ -9,11 +9,34 @@
 
 #include <cstdint>
 
+#include "facade/facade_types.h"
+
 extern "C" {
 #include "redis/sds.h"
 }
 namespace dfly {
 
+template <typename O>
+static std::vector<long> ExpireElements(void* ptr, const facade::CmdArgList values,
+                                        uint32_t ttl_sec) {
+  O* owner = (O*)ptr;
+  std::vector<long> res;
```

Comment:
```suggestion
template <typename DenseSet>
static std::vector<long> ExpireElements(DenseSet* owner, const facade::CmdArgList values,
                                        uint32_t ttl_sec) {
  std::vector<long> res;
```

Reply:
that way you don't need:

1. To cast a `void*` to the respective type
2. You can now call `ExpireElements(arg1, arg2)` without the `<>` syntax. (that is, let the compiler automatically deduce this)